python/cudf/cudf/core/column/column.py

```diff
@@ -321,6 +327,38 @@ def to_array(self, fillna=None):
         """
         return self.to_gpu_array(fillna=fillna).copy_to_host()
 
+    def fill(self, fill_value, begin=0, end=-1, inplace=False):
+
+        if end == -1:
+            end = self.size
+
+        assert end <= self.size
+        assert begin <= end
+
+        if begin == end or begin == self.size:
+            return self if inplace else self.copy()
```

Comment:
In Python typically negative indices are used to indicate distance from the end where it would be good to mirror the semantics of that behavior here. See: https://github.com/rapidsai/cudf/blob/branch-0.13/python/cudf/cudf/core/frame.py#L131-L140